jenkins-job-builder/builders.yml

```diff
@@ -144,10 +144,10 @@
     name: copy-pw-files
     builders:
         - shell: |
-            mkdir -p $HOME/.ipa/alias
+            rm -rvf $HOME/.ipa
+            mkdir $HOME/.ipa
             echo $FREEIPACI_PASSWORD > $HOME/.ipa/.dmpw
-            sudo cp /etc/httpd/alias/*.db $HOME/.ipa/alias
-            sudo cp /etc/httpd/alias/pwdfile.txt $HOME/.ipa/alias/.pwd
```

Comment:
This line is needed. The testing framework expects the data from pwdfile.txt in ~/.ipa/alias/.pwd